migrations/versions/195b6eb2fa1b_add_knewbie_id.py

```diff
@@ -0,0 +1,102 @@
+"""Add Knewbie ID
```

Comment:
Would have to reinitialize db since some tables were added. These versions can all be deleted.